cassovary-core/src/main/scala/com/twitter/cassovary/graph/ArrayBasedDynamicDirectedGraph.scala

```diff
@@ -46,7 +46,8 @@ class ArrayBasedDynamicDirectedGraph(val storedGraphDir: StoredGraphDir)
 
   def this(iterableSeq: Seq[Iterable[NodeIdEdgesMaxId]],
            storedGraphDir: StoredGraphDir) {
-    this(iterableSeq.flatten , storedGraphDir)
+    // Use view to avoid eagerly reading all edges into memory
+    this(iterableSeq.view.flatten.toIterable, storedGraphDir)
```

Comment:
Yes, I don't know why, but otherwise it doesn't compile.  The type without the toIterable seems to be
`scala.collection.SeqView[Iterable[com.twitter.cassovary.graph.NodeIdEdgesMaxId],Seq[Iterable[com.twitter.cassovary.graph.NodeIdEdgesMaxId]]]#Transformed[com.twitter.cassovary.graph.NodeIdEdgesMaxId]`
It's quite a type :)